package.json

```diff
@@ -52,8 +58,10 @@
     "nyc": "^14.0.0",
     "opener": "^1.5.1",
     "prettier": "^2.3.1",
+    "pug-lexer": "^5.0.1",
     "rimraf": "^2.6.3",
     "rollup": "^1.1.2",
+    "rollup-plugin-commonjs": "^10.1.0",
```

Comment:
Needed cause pug-lexer is a commonjs package and there are errors when trying to import something from it without this plugin